src/main/java/com/denys/toukdemo/dto/ReservationUnit.java

```diff
@@ -0,0 +1,41 @@
+package com.denys.toukdemo.dto;
+
+public class ReservationUnit {
+
+    private Integer row;
```

Comment:
co będzie jak podam tu null?

Reply:
> co będzie jak podam tu null?

też trzeba dodać walidację not null w tych klasach. mój błąd